apis/rds/v1alpha1/zz_doc.go

```diff
@@ -0,0 +1,6 @@
+// +kubebuilder:object:generate=true
```

Comment:
In the latest `main` branch, this file has the necessary license header. I'd suggest pulling latest ACK and run generation again.